LibraryManagement.cpp

```diff
@@ -2,129 +2,206 @@
 #include <string>
 #include <conio.h>
 #include <cstdlib>
+#include <fstream>
 #include <ctime>
 #include "book.h"
 #include "user.h"
 #include "bookmark.h"
 #include "Menu.h"
 
 
+
 using namespace std;
 
-bool isLoggedIn() {
-    static bool loggedIn = false;
-    return loggedIn;
+bool verifyUser(const string& username, const string& password) {
+    ifstream file("users.txt");
+    if (file.is_open()) {
+        string name, pass;
+        while (file >> name >> pass) {
+            if (name == username && password == pass) {
+                file.close();
+                return true;
+            }
+        }
+        file.close();
+    }
+    return false;
+}
+
+bool registerUser(const string& username, const string& password) {
+    ofstream file("users.txt", ios::app);
+    if (file.is_open()) {
+        file << username << " " << password << endl;
+        file.close();
+        return true;
+    } else {
+        return false;
+    }
 }
 
 Menu menu;
 Bookmark bookMark;
 
 int main() {
-	bool quit{false};
-int choose;
+	bookMark.addDefaultBooks();
```

Comment:
why did you resolved conversation? Use constructor creation here instead of calling specific method.

Example: Bookmark bookmark = new Bookmark() -> let constructor create an object and load all needed books